### PR TITLE
cypress test for edit bounty

### PR DESCRIPTION
**Pull request by Vayras**
## Describe your changes
Added Cypress Test with this PR for edit bounty, as mentioned in issue:#380

## Type of change
Test 

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have rebased to master

### CODE REVIEW

**Comment by elraphty** [comment]
@Vayras the cypress tests are failing, did you try running is locally?

**Comment by Vayras** [comment]
> @Vayras the cypress test is failing, did you try running them locally?

@elraphty I'm facing trouble trying to run Cypress on apple silicon, I'm currently reinstalling Cypress and will try to fix this

**Comment by elraphty** [comment]
> > @Vayras the cypress test is failing, did you try running them locally?
> 
> @elraphty I'm facing trouble trying to run Cypress on apple silicon, I'm currently reinstalling Cypress and will try to fix this

Okay, nice

**Comment by Vayras** [comment]
@elraphty fixed cypress test on this one, can you please review this, Thanks

File: cypress/e2e/6_editBountyFromModal.cy.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    cy.wait(1000);
+
+    const newTitle = 'Edited Bounty Title';
+    const newDescription = 'This bounty has been edited';
+
+    cy.get('.inputText').eq(0).clear({ force: true }).type(newTitle);
+    cy.wait(1000);
+    cy.get('[data-testid="checktextarea"]').clear().type(newDescription);
+    cy.wait(1000);
+
```
[the remark this answers is not in the document]
Comment (Vayras) [reply cypress/e2e/6_editBountyFromModal.cy.ts:41]: @elraphty done

**Pull request merged**